### PR TITLE
Remove Integration button text changed

### PR DESCRIPTION
**Pull request by Krish-bhardwaj**
## Description

Fix https://github.com/gitpod-io/gitpod/issues/7521

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@Krish-bhardwaj: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankeromnes** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankeromnes` in a comment when ready.

Associated issue: *#7521*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Krish-bhardwaj** [comment]
Hi i would request you to check and merge my PR

**Comment by Krish-bhardwaj** [comment]
Hi @gtsiolis  can you plz check my PR it is a very small change

**Comment by Krish-bhardwaj** [comment]
> Thanks for your contribution! Hi please use our PR template @Krish-bhardwaj

This was a very small change i have mentioned everthing requested to check my PR

**Comment by gtsiolis** [comment]
OK, looks pretty straight forward.

Looking at this now! 👀

**Review by gtsiolis — APPROVED** [review]
Changes look good and pretty straight forward, @Krish-bhardwaj! 🔮 

Let's merge this!

Thanks also @iQQBot for helping here! 🏀

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: c8d2251687a0c55eeed98aad677cfa3655517822</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7560#pullrequestreview-850939759" title="Approved">gtsiolis</a>*

Associated issue: *#7521*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
